tldw_Server_API/app/main.py

```
@@ -2694,6 +2694,24 @@ async def _ephemeral_cleanup_loop():
+    # Study-pack Jobs worker
+    try:
+        import asyncio as _asyncio
+
+        _enabled = _should_start_worker("STUDY_PACK_JOBS_WORKER_ENABLED", "flashcards")
+        if _enabled:
+            from tldw_Server_API.app.services.study_pack_jobs_worker import (
+                run_study_pack_jobs_worker as _run_study_pack_jobs,
+            )
+
+            study_pack_jobs_stop_event = _asyncio.Event()
+            study_pack_jobs_task = _asyncio.create_task(_run_study_pack_jobs(study_pack_jobs_stop_event))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)